ruma-events/src/policy/rule.rs

```diff
@@ -0,0 +1,62 @@
+//! Modules and types for events in the *m.policy.rule* namespace.
+
+use std::fmt::{Display, Formatter, Result as FmtResult};
+
+use serde::{Deserialize, Serialize};
+
+pub mod room;
+pub mod server;
+pub mod user;
+
+/// The payload for policy rule events.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct PolicyRuleEventContent {
```

Comment:
This should also have a feature-gated non-exhaustive attribute and a constructor.